### DISCUSSION TITLE
Possible bug

### DISCUSSION THREAD

**Discussion by SzymonSliwinski** [ngrx/platform#3862] [General]
Hi 
I'm not sure but i think i've found a bug or maybe I'm doing something wrong. 
For versions > 14.2.0 I have found strange behavior in memory profiler, more info here.
https://stackoverflow.com/questions/76113632/angular-constantly-growing-store/76122963#76122963

Can you tell me if this is a bug?

**Comment by e-oz** [comment]
Try to reproduce it on StackBlitz

**Reply by SzymonSliwinski** [reply]
Here's [code](https://stackblitz.com/edit/ngx-14-title-demo-4cmaou?file=src%2Findex.html,src%2Fapp%2Fapp.component.ts)
